WORKSPACE

```diff
@@ -141,10 +141,10 @@ http_file(
 
 http_file(
     name = "virtio_win_image",
-    sha256 = "b8a4bc66835c43091a85d35a10b59bd8b1b62b55ea9f02ec754f68bd32e82c0e",
+    sha256 = "d5b5739cf297f0538d263e30678d5a09bba470a7c6bcbd8dff74e44153f16549",
     urls = [
-        "https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/archive-virtio/virtio-win-0.1.217-1/virtio-win-0.1.217.iso",
-        "https://storage.googleapis.com/builddeps/b8a4bc66835c43091a85d35a10b59bd8b1b62b55ea9f02ec754f68bd32e82c0e",
+        "https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/archive-virtio/virtio-win-0.1.248-1/virtio-win-0.1.248.iso",
+        "https://storage.googleapis.com/builddeps/d5b5739cf297f0538d263e30678d5a09bba470a7c6bcbd8dff74e44153f16549",
```

Comment:
I think it is okay to drop this line. It will be re-added by the automation once the file was mirrored.

Reply:
But it can stay right? :D

Reply:
It could lead to errors when trying to access the mirror, but I don't think it makes things worse when the first mirror is already failing.

Reply:
this would make the bot skip the mirroring (thinking it already exists) and thus when the main mirror dies, we will not have backup

Reply:
Good/bad to know. Is there a way to make the bot more robust?